metadata.go

```diff
@@ -23,23 +23,23 @@ func (a *Application) ParseAllMetadata() error {
 
 // SetMetadataString for a given instance before register
 func (ins *Instance) SetMetadataString(key, value string) {
-	if ins.Metadata.parsed == nil {
-		ins.Metadata.parsed = map[string]interface{}{}
+	if ins.Metadata.Parsed == nil {
```

Comment:
Given that we can _set_ entries in this field, "Parsed" doesn't match its use. "Parsed" implies that we only _receive_ these values.

Again, consider renaming it to "Map", "Fields", "Values", or something similar.